compiler-core/src/javascript/tests/snapshots/gleam_core__javascript__tests__externals__external_type_typescript.snap

```diff
@@ -2,6 +2,9 @@
 source: compiler-core/src/javascript/tests/externals.rs
 expression: "pub type Queue(a)\n\n@external(javascript, \"queue\", \"new\")\npub fn new() -> Queue(a)\n"
 ---
-export type Queue$<I> = any;
+declare const __brand_Queue: unique symbol;
+type Queue = { [__brand_Queue]: void };
```

Comment:
This is incorrect, it is not an accurate description of the type.

We will not accept any inaccurate type definitions, even if the inaccuracy is a popular TS inference hack.

Reply:
Hmm, if the requirement is to have a truly accurate TypeScript type then I think the author would need to provide it manually, perhaps via a type-based external like this:

```gleam
@external(typescript, "my_ffi.d.mts", "Dict<A, B>")
pub type Dict(A, B)
```

I'm not meaning to propose the above as a new language feature though, just thinking out loud.

In the meantime, changing the `any` to `unknown` helps somewhat. I've reduced this PR to only do that as detailed in the reply below.